done coursework

Delegant opened the pull request.

Commit:
done coursework

OlgaLugacheva commented on src/main/java/pro/sky/telegrambot/model/Notification.java:
```diff
@@ -0,0 +1,49 @@
+
+    public enum NotificationStatus {
+        SCHEDULED,
+        SENT,
```
неиспользуемое значение - нигде не меняется

OlgaLugacheva commented on src/main/java/pro/sky/telegrambot/model/Notification.java:
```diff
@@ -0,0 +1,49 @@
+    private LocalDateTime notificationDate;
+
+    @Enumerated(EnumType.STRING)
+    private NotificationStatus status = NotificationStatus.SCHEDULED;
```
если нигде не используется, стоит убрать

OlgaLugacheva commented on src/main/java/pro/sky/telegrambot/model/Notification.java:
```diff
@@ -0,0 +1,49 @@
+
+    private String notificationMessage;
+
+    private LocalDateTime sentDate;
```
нигде не обновляется это поле - если мы его не используем, стоит убрать

Commit:
drop files from /.idea/
drop not used fields from Notification class
drop not used columns from Notification table
